gatsby-node.js

```diff
@@ -34,16 +34,22 @@ exports.onPostBuild = async function(
   setStatus(activity, `Loaded algolia hash index ${Object.keys(aIndex).length}`);
 
   const hashObject = (queryIndex, obj) => {
-    let hash = crypto.createHash(`md5`).update(JSON.stringify(obj)).digest(`hex`);
-    let oldHash = aIndex[queryIndex] && aIndex[queryIndex][obj.id]
+    const ID = obj.id || obj.objectID;
```

Comment:
I think in the context of Algolia it's not possible to do a proper diff if objectID isn't stable, so let's force that.

Reply:
You thinking only use objectID?

Reply:
unless you have a use case for not using solely objectID

Reply:
Was more for people too lazy to swap out objectID for id, because they might use `id` in the Algolia results, but if it is a duplicate of objectID then it might just be dumb.

But I do agree, I'll update to only use objectID. Should we throw some error if it does not exist?

Reply:
Yes, an error seems appropriate here